collector/zfs.go

```diff
@@ -0,0 +1,109 @@
+package collector
```

Comment:
Please include the same copyright/license header here as in all the other Node Exporter files. Same for the other Go files here.